reporters/stenographer/stenographer.go

```diff
@@ -8,9 +8,11 @@ package stenographer
 
 import (
 	"fmt"
+	"io"
 	"runtime"
 	"strings"
 
+	"github.com/mattn/go-colorable"
```

Comment:
hmmm.  i see why this is needed, but i think  this just a single file ? if so i wonder if ginkgo should really just carry it, ... b/c as of now there is no support or need for GoDep.